TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Shooter.java

```diff
@@ -0,0 +1,69 @@
+package org.firstinspires.ftc.teamcode;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorSimple;
+import com.qualcomm.robotcore.hardware.Gamepad;
+
+@TeleOp(name = "Shooter", group = "LinearOpMode")
+public class Shooter extends LinearOpMode {
+    @Override
+    public void runOpMode() throws InterruptedException {
+        DcMotor leftShooterMotor = hardwareMap.dcMotor.get("leftShooterMotor");
+        DcMotor rightShooterMotor = hardwareMap.dcMotor.get("rightShooterMotor");
+
+        rightShooterMotor.setDirection(DcMotorSimple.Direction.REVERSE);
+
+        waitForStart();
+
+        //Shooters start at 10% power
+        double leftShooterPower = .1;
+        double rightShooterPower = .1;
+
+        if (isStopRequested()) return;
+
+        Gamepad currentGamepad1 = new Gamepad();
+        Gamepad previousGamepad1 = new Gamepad();
+
+        while (opModeIsActive()) {
+            //The Y button is for shooting
+            //Use the Dpad to change the speed of the motors
+            boolean shoot = gamepad1.y;
+            boolean more = gamepad1.dpad_up;
+            boolean less = gamepad1.dpad_down;
+
+            previousGamepad1.copy(currentGamepad1);
+            currentGamepad1.copy(gamepad1);
+
+            if (shoot == true) {
+                leftShooterMotor.setPower(leftShooterPower);
+                rightShooterMotor.setPower(rightShooterPower);
+            }else {
+                leftShooterMotor.setPower(0);
+                rightShooterMotor.setPower(0);
+            }
+
+            //Shooter power increases by 10% everytime Dpad up is pressed
+            if (currentGamepad1.dpad_up && !previousGamepad1.dpad_up) {
```

Comment:
Don't have to do this in this PR but we should create some setter functions like... 

```
public void increasePower() {
                leftShooterPower = leftShooterPower + 0.1;
                rightShooterPower = rightShooterPower +0.1;
}
```